src/devices/duplotrainbasespeedometer.ts

```diff
@@ -15,6 +15,10 @@ export class DuploTrainBaseSpeedometer extends Device {
     }
 
     public receive (message: Buffer) {
+        if (this.hub.autoParse) {
```

Comment:
Also the check could be changed from `this.hub.autoParse` to `this.autoParse` with the getter method.